src/Symfony/Component/VarDumper/Caster/ReflectionCaster.php

```diff
@@ -83,13 +83,13 @@ public static function castGenerator(\Generator $c, array $a, Stub $stub, bool $
         // Cannot create ReflectionGenerator based on a terminated Generator
         try {
             $reflectionGenerator = new \ReflectionGenerator($c);
+
+            return self::castReflectionGenerator($reflectionGenerator, $a, $stub, $isNested);
         } catch (\Exception $e) {
             $a[Caster::PREFIX_VIRTUAL.'closed'] = true;
-
-            return $a;
```

Comment:
Why not keep this line and remove the `return` statement below entirely instead (IMO makes the diff more understandable)?